bigquery/README.md

```diff
@@ -70,3 +70,8 @@ pip install -r requirements-to-freeze.txt
 pip freeze | sort -f | sed 's/^indexer-util.*/\.\/indexer_util/g' > requirements.txt
 deactivate
 ```
+
+### Troubleshooting
```

Comment:
Do you know if this is related to us removing the vm.max_map_counts requirement? Does it work if you set that?

Reply:
Okay, looks like it is a docker issue then. Thanks for double checking!